hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/has/isauthorizedraw/IsAuthorizedRawCall.java

```diff
@@ -62,7 +62,12 @@ public class IsAuthorizedRawCall extends AbstractCall {
 
     // FUTURE: Gas for system contract method calls needs to be a) determined by measurement of
     // resources consumed, and b) incorporated into the fee schedule
-    private static final long HARDCODED_GAS_REQUIREMENT_GAS = 1_500_000L;
+
+    // Gas charge for an ED key - (rough) estimate of resources used (FUTURE: to be refined)
+    private static final long HARDCODED_ED_GAS_REQUIREMENT_GAS = 1_500_000L;
+
+    // Gas charge for an EC key - this is what Ethereum charges for ECRECOVER precompile
+    private static final long HARDCODED_EC_GAS_REQUIREMENT_GAS = 3_000;
```

Comment:
is it possible to use something like `gasCalculator().getEcrecPrecompiledContractGasCost()` so that if this value changes we are covered?

Reply:
Done.